src/Rules.php

```diff
@@ -5,72 +5,35 @@
 namespace Yiisoft\Validator;
 
 use Yiisoft\Validator\Rule\Callback;
-use Yiisoft\I18n\TranslatorInterface;
 
 /**
  * Rules represents multiple rules for a single value
  */
-class Rules
+final class Rules
```

Comment:
Why?

Reply:
Do you have a use-case for extending it?